eslint.config.js

```diff
@@ -0,0 +1,20 @@
+import standardConfig from './src/standardConfig.js'
+
+export default [
+  ...standardConfig({
+    node: true,
+    prettier: true,
+    sortImports: true,
+    typescript: true
```

Comment:
I'm reading it's a good practice to define `tsconfigRootDir: import.meta.dirname` so `@typescript-eslint/parser` knows where to look for tsconfig.json

If not provided, defaults to undefined, which interferes with TS ESLint's new "project service" feature.

Alternatively, we can fix the issue in `standardConfig.js` itself

Reply:
They fixed this. Recent versions of @typescript-eslint automatically locate the appropriate tsconfig.json, so it "Just Works".

The `import.meta.dirname` property isn't available in Node 18, so we'd have to do `dirname(fileUrlToPath(import.meta.url))`, which is far more annoying. Given this, it's simpler just to use the defaults, already do the right thing.